lib/fitfree.gi

```diff
@@ -108,7 +108,11 @@ local ffs,hom,U,rest,ker,r,p,l,i,depths;
   # transfer the IndicesEANormalSteps
   if ffs.pcgs<>ipcgs and HasIndicesEANormalSteps(ffs.pcgs) then
     U:=IndicesEANormalSteps(ffs.pcgs);
-    r:=ipcgs!.depthsInParent;
+    if IsBound(ipcgs!.depthsInParent) then
+      r:=ipcgs!.depthsInParent;
+    else
+      r:=List(ipcgs,x->DepthOfPcElement(ffs.pcgs,x));
+    fi;
```

Comment:
Hmm, so didn't the old code sometimes run into errors? Are there "cheap" examples that show this? In any case, this sounds like a bug fix to me, so perhaps relevant for the release notes after all? Unless it is in code that is undocumented and not normally triggered by user inputs, perhaps... ? I can't judge that right now.

Reply:
The code relied on a side-effect, a `depthsInParent` entry, being bound which is not always the case. The example I had was with a matrix group over Z/composite number and the matgrp package.